### PR TITLE
Lambda build server37 rebased

### PR DESCRIPTION
**Pull request by sandyhider**
Updated salt scripts for python3.7 and lambda-build-server.  
Added missing roles and policies

### CODE REVIEW

File: docs/Testing.md [hunk 1]
```diff
@@ -1,6 +1,71 @@
...
+
+### Copy up dynamo information in spdb for testing
+Now the spdb is a pypi installable it doesn't include the contents of 
+spdb/spdb/spatialdb/dynamo/
+You need to tar up the contents of that directory and use ssh-tunnel to copy it to the endpoint for testing
+```bash
+$ cd spdb/spdb/spatialdb
+$ tar czf /path/to/spdb-spatialdb-dynamo.tar.gz ./dynamo
+$ cd spdb
+$ tar rzf /path/to/spdb-spatialdb-dynamo.tar.gz ./requirements-test.txt
```
Comment (movestill) [inline docs/Testing.md:13]: Probably easier to just `pip install -r requirements-text.txt` on the endpoint?

File: salt_stack/salt/lambda-dev/init.sls [hunk 2]
```diff
@@ -4,21 +4,14 @@
...
     - node
-
-python36:
-    pkg.installed:
-        - pkgs:
-            - python36
-            - python36-pip
-            - python36-virtualenv
-
+    - python.python37
```
Comment (movestill) [inline salt_stack/salt/lambda-dev/init.sls:14]: What OS are we running on the lambda build server now? Is it possible to get 3.7 with the OS' package manager?

Comment (sandyhider) [reply salt_stack/salt/lambda-dev/init.sls:14]: It's amazon linux.  I looked they didn't have a 3.7

File: docs/Testing.md (hunk 1 above)
Comment (sandyhider) [reply docs/Testing.md:13]: You mean run that as part of the packer script?  I like that idea, if that is what you mean.

**Pull request merged**